modules/ROOT/pages/provisioning-qemu.adoc

```diff
@@ -47,16 +47,16 @@ IGNITION_DEVICE_ARG="-fw_cfg name=opt/com.coreos/config,file=${IGNITION_CONFIG}"
 IGNITION_DEVICE_ARG="-drive file=${IGNITION_CONFIG},if=none,format=raw,readonly=on,id=ignition -device virtio-blk,serial=ignition,drive=ignition"
 
 qemu-kvm -m 2048 -cpu host -nographic -snapshot \
-  -drive if=virtio,file=${IMAGE} ${IGNITION_DEVICE_ARG} \
+  -drive if=virtio,file="${IMAGE}" "${IGNITION_DEVICE_ARG}" \
```

Comment:
```suggestion
  -drive if=virtio,file="${IMAGE}" ${IGNITION_DEVICE_ARG} \
```

Looks like this one should be split on space

Reply:
Should be fixed now.